source/includes/steps-fle-convert-to-a-remote-master-key.yaml

```diff
@@ -0,0 +1,73 @@
+title: Create an AWS IAM User
+ref: create-an-aws-iam-user
+content: |
+  Create a new programmatic IAM user to use in CSFLE-enabled clients.
+  The user will encrypt and decrypt the remote master key and must have
+  full ``List`` and ``Read`` permissions for the KMS service.
+
+  .. admonition:: Client IAM User Credentials
+     :class: note
+     
+     The CSFLE-enabled client takes the IAM User's :guilabel:`Access Key
+     ID` and :guilabel:`Secret Access Key` as configuration values. Note
+     these down for later when we reconfigure the client.
+---
+title: Create the Master Key
+ref: create-the-master-key
+content: |
+  In AWS KMS, generate a new master key. The key's name and description
+  don't affect the functionality of CSFLE but should describe that it's
+  for the CSFLE-enabled client.
+  
+  In the :guilabel:`Usage Permissions` step of the key generation
+  process, select the newly created IAM User with full KMS ``List`` and
+  ``Read`` permissions. This allows the user to encrypt and decrypt the
+  new master key.
+
+  .. important::
+     
+     The new client IAM User *should not* have administrative permissions
+     for the master key.
+---
+title: Specify the AWS KMS Provider Credentials
+ref: specify-the-aws-kms-provider-credentials
+content: |
+  Unlike the local KMS provider, the AWS KMS provider does not accept
+  the master key directly from the client configuration code. Instead,
+  it accepts the :guilabel:`Access Key ID` and :guilabel:`Secret Access
+  Key` of the IAM user with permission to encrypt and decrypt the master
+  key.
+
+  Update the KMS Provider configuration in CSFLE-enabled client creation code:
+  
+  .. tabs::
+     
+     .. tab:: Java
+        :tabid: use-cases-fle-guide-java
+        
+        .. code-block:: java
+           :emphasize-lines: 4-6
+           
+           String awsAccessKeyId = "<IAM User Access Key ID>";
+           String awsSecretAccessKey = "<IAM User Secret Access Key>";
+           Map<String, Map<String, Object>> kmsProviders = new HashMap<String, Map<String, Object>>() {{
+             put("aws", new HashMap<String, Object>() {{
+               put("accessKeyId", awsAccessKeyId);
+               put("secretAccessKey", awsSecretAccessKey);
+             }});
+           }};
+---
+title: Create a New Data Key and JSON Schema
+ref: create-a-new-data-key-and-json-schema
+content: |
+  The development data key was generated from a local master key, so you
```

Comment:
Great inclusion here. More of a non-blocking question, should we make this an admonition for lazy readers? "Once you are using a remote master key, the data keys you were using in testing **will not** work"?

Reply:
Not sure about this, let's revisit in our unification pass